### ISSUE TITLE
[docs] F.softmax _stacklevel in argspec, but not described in Parameters section

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#36524]
https://pytorch.org/docs/master/nn.functional.html?highlight=softmax#torch.nn.functional.softmax

**Comment by vadimkantorov** [comment]
I think this is related to the warning produced when `dim` is not  unspecified. If that warning /  unspecified `dim` behavior is killed, this `_stacklevel` mysterious arg also can go

**Comment by asanakoy** [comment]
what is `_stacklevel` by the way?

**Comment by vadimkantorov** [comment]
It is somehow related to Python's warning suppression mechanism

**Comment by vadimkantorov** [comment]
I propose to remove these args before 3.0.0 :)

And maybe make the functional wrappers just be `F.softmax = torch.softmax` ?
